.env.test

```diff
@@ -1,9 +1,4 @@
 RAILS_ENV=test
-DATABASE_APP_DB_TEST=warehouse_app_test
-WAREHOUSE_DATABASE_DB_TEST=warehouse_test
-HEALTH_DATABASE_DB_TEST=health_test
-REPORTING_DATABASE_DB_TEST=reporting_test
-WAREHOUSE_DATABASE_ADAPTER=postgis
 FQDN=hmis-warehouse.test
```

Comment:
Why are we removing these?

Reply:
I had a `.env` file locally with identical settings and no secrets. I use `.env.local` most of the time which overrode all of them so I thought this project had a `.env` committed with sensible defaults and had forgotten that is/was part of the initial setup scripts.